ddd-koans/Value Object/Introduction/src/Main.kt

```diff
@@ -0,0 +1,2 @@
+fun main() {
+}
```

Comment:
maybe at least some comment here like `nothing to do here, go to next task` or sth similar

Reply:
already fixed